schedule-2019-03-internal.md

```diff
@@ -0,0 +1,95 @@
+# Instructor Training (Internal, March 2019)
+
+Copyright 2019, RStudio Inc.
+All material made available under the [Creative Commons Attribution-NonCommercial 4.0](https://creativecommons.org/licenses/by-nc/4.0/) license.
+
+## Instructors
+
+- Garrett Grolemund
+- Greg Wilson
+
+## Day 1
+
+1.  Getting Started (0:15/09:15)
+
+1.  Two Ways to Teach (0:45/10:00)
+    -   <./keynote/10-two-ways-to-teach.key>
+    -   Comparison with movie/play (add comedy improv)
+    -   Tell participants there's lots of research (much of which people don't know/act on)
+    -   Explain what active learning is (creates a real-time feedback loop in class)
+    -   Introduce cognitive architecture diagram (revisit it several times)
+        -   Teaching is "load into short-term memory and keep it there long enough to be transferred"
+    -   Bloom's Taxonomy
+        -   Google isn't enough if you don't know what to search for or how to recognize a useful answer when you find one
+        -   GG added "awareness" to the bottom of the Bloom hierarchy since it's the most realistic goal for a workshop like this
+    -   Exercise: classify three learning objectives by level
```

Comment:
I suggest moving Bloom's Taxonomy to Novice/Competent Practitioner/Expert. Use it there as a way to demonstrate how more complex mental models facilitate increasingly complex skills. Then change the proposed exercise to one very targeted to the students: come up with a short list of questions in your area of expertise that you could use to quickly tell whether you are talking to an expert, novice, or someone in between.

Reply:
I've moved Bloom's right to the end of day 2 when we talk about the Divio Model - after our discussions, I don't think it's important enough to get air time during the body of the training, and if we pair it with the "unconscious incompetence - unconscious competence" progression, it'll help give them a framework for critiquing R Markdown.